src/components/button/button.tsx

```diff
@@ -20,7 +21,7 @@ export class Button {
 
   render() {
     return (
-      <ion-button shape="round" data-testid={this.dataTestId} expand={this.expand} disabled={this.disabled} onClick={this.clickHandler} {...this.props}>
+      <ion-button shape="round" size={this.size || 'default'} data-testid={this.dataTestId} expand={this.expand} disabled={this.disabled} onClick={this.clickHandler} {...this.props}>
```

Comment:
@charlesoma I suggest setting the default value on the property itself and not detaching it. This keeps things slim and consistent (in case this property is used elsewhere).

```
@Prop() size?: 'large' | 'small' | 'default' = 'default';
```